citi-onboarding/Aqua-Jr
# Feature/banner front

# What i did:
- Created the web version to the banner
- Created the mobile version to the banner.
- Hotfix in the newsletter
- Hotfix in the Testimony
# How to test:
1. See if the web banner is equal/simmilar to the mockup
2. See if the mobile banner is equal/simmilar to the mockup
3. Check if anything is broken with:
3.1 Newsletter
3.2 Testimony
